test/images/scripts/lib/debian.install

```diff
@@ -46,6 +46,9 @@ if [ -n "$do_build" ]; then
     tar -xzf cockpit-*.tar.gz
     ( cd cockpit-*/
       cp -rp tools/debian debian
+      # Hack: Remove PCP build dependencies while pcp is not in testing
+      # (https://tracker.debian.org/pcp)
+      sed -i '/libpcp.*-dev/d' debian/control
```

Comment:
... oh, that of course means that we have to drop the pcp build deps from the actual upload. If we can do this more elegantly someplace else, I'm all for it of course.

A different approach would be to use build dependency alternatives like `pcp-dev | aspell-doc` so that the builders install the (harmless) aspell-doc if pcp is not available. But this is really quite a hack and not even guaranteed to work on Debian builders (I know that Ubuntu's builders try alternate build dependencies, but Debian's didn't in the past), so I'd rather do the uploads in a way that the uploaded source package has the pcp build deps removed.